### PR TITLE
add-return-types

### PR DESCRIPTION
**Pull request by beattie282**

### CODE REVIEW

File: src/Connections/Curl.php [hunk 1]
```diff
@@ -51,7 +51,7 @@ public function post(string $url, array $postOptions, array $getOptions = [])
         return json_decode(curl_exec($this->handle));
     }
 
-    public function delete(string $url, array $postOptions, array $getOptions = [])
+    public function delete(string $url, array $postOptions, array $getOptions = []): mixed
```
Comment (exussum12) [inline src/Connections/Curl.php]: Is this mixed ?

File: src/CodeManager.php [hunk 2]
```diff
@@ -3,10 +3,9 @@
 
 interface CodeManager
 {
-    /** @return BranchInfo[] */
```
Comment (exussum12) [inline src/CodeManager.php:6]: This should stay ?

File: src/Connections/Curl.php (hunk 1 above)
Comment (beattie282) [reply src/Connections/Curl.php]: I believe so. Its returning JSON like the post and get functions.

**Review by richardhughes — APPROVED** [review]

File: src/Connections/Curl.php (hunk 1 above)
Comment (exussum12) [reply src/Connections/Curl.php]: Ill probably firm them both up into proper objects at some point

**Pull request merged**